src/Repository/PaypalAccountRepository.php

```diff
@@ -50,6 +50,7 @@ public function getOnboardedAccount()
         return new PaypalAccount(
             $this->getMerchantId(),
             $this->getMerchantEmail(),
+            $this->getMerchantCountry(),
             $this->getMerchantEmailStatus(),
             $this->getPaypalPaymentStatus(),
             $this->getCardHostedFieldsStatus()
```

Comment:
Constructor params moved
```suggestion
            $this->getMerchantEmailStatus(),
            $this->getPaypalPaymentStatus(),
            $this->getCardHostedFieldsStatus()
            $this->getMerchantCountry(),
```